var/spack/repos/builtin/packages/jogl/package.py

```diff
@@ -27,9 +27,7 @@ class Jogl(Package):
     # Xfree86 Video mode extentions is deprecated
     patch("noxf86vm.patch")
 
-    phases = ["edit", "build", "install"]
```

Comment:
I'll remove this commit. This was added before 9981a5d4e127512cf00126bc0939cb1d292e1b56 Since we have that, I think there is no need to change the recipe in this PR.

Reply:
> Maybe a CustomPhasesPackage metaclass would be useful long-term?

If we make clear in the docs (or enforce in code, if possible) that this is the only class to be used when custom phases are considered helpful, then I think it would be an interesting idea.

Reply:
Commit removed from the PR.